api/v1alpha1/limitador_types.go

```diff
@@ -95,6 +94,8 @@ type Limitador struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:XValidation:rule="(!has(self.storage) || !has(self.storage.disk)) || (!has(self.replicas) || self.replicas < 2)",message="disk storage does not allow multiple replicas"
```

Comment:
I'm not sure this set of rules is valid... because the first branch of the logical _OR_ will escape the main _OR_ when _True_ and never evaluate the second, which is the asserting the replica count

Reply:
Sorry, my bad, the second half of the first branch is negated `!has(self.storage.disk)`